README.md

```diff
@@ -1,151 +1,54 @@
-
-- Python 3.7.1 or higher
-
-If you do not have a version of Python greater than 3.7.1, it is recommended to use `pyenv` to be able to easily use and switch between multiple Python versions.
-
-- [`pyenv`](https://github.com/pyenv/pyenv)
-
-It is recommended that you install the `poetry` dependency manager if you are a current (or potential) `schematic` contributor or a DCC admin managing installations of the [Data Curator App](https://github.com/Sage-Bionetworks/data_curator/).
-
-- [`poetry`](https://github.com/python-poetry/poetry)
-
-
-**Important**: Make sure you are a registered and certified user on [`synapse.org`](https://www.synapse.org/), and also have all the right permissions to download credentials files in the following steps. Contact your DCC liaison to request for permission to access the credentials files.
-
-## 1.3. Package Setup Instructions
-
-### 1.3.1. Clone Project Repository
-
-Since the package isn't available on [`PyPI`](https://pypi.org/) yet, to setup the package you need to `clone` the project repoository from GitHub by running the following command:
 
-```bash
-git clone --single-branch --branch develop https://github.com/Sage-Bionetworks/schematic.git
-```
-
-### 1.3.2. Virtual Environment Setup
-
-```python
-python -m venv .venv  # create a virtual environment
-```
-
-```bash
-source .venv/bin/activate # activate the `venv` virtual environment
-```
-
-### 1.3.3. Install Dependencies
-
-After cloning the `schematic` project from GitHub and setting up your virtual environment:
-```bash
-cd schematic  # change directory to schematic
-git checkout develop  # switch to develop branch of schematic 
-poetry build # build source and wheel archives
-pip install dist/schematicpy-0.1.11-py3-none-any.whl  # install wheel file
-```
-
-### 1.3.4. Obtain Credentials File(s)
-
-```bash
-synapse get syn21088684 # download copy of credentials.json file
-```
-
-The `credentials.json` file is required when you are using [`OAuth2`](https://developers.google.com/identity/protocols/oauth2) to authenticate with the Google APIs.
-
-For details about the steps involved in the `OAuth2` [authorization flow](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/utils/google_api_utils.py#L18), refer to the `Credentials` section in the [`docs/details`](https://github.com/Sage-Bionetworks/schematic/blob/develop/docs/details.md#credentials) document.
-
-```bash
-synapse get syn24214983 # download copy of schematic_service_account_creds.json file
-```
-
-Use the `schematic_service_account_creds.json` file for the service account mode of authentication (_for Google services/APIs_).
-
-Note: The `Selection Options` dropdown which allows the user to select multiple values in a cell during manifest annotation [does not work](https://developers.google.com/apps-script/api/concepts) with the service account mode of authentication.
-
-### 1.3.5. Fill in Configuration File(s)
+- [Schematic](#schematic)
```

Comment:
[Nit:](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.) I would remove this line and de-indent the rest.

Reply:
So this markdown extension I'm using called "Markdown All in One" for VSCode generates the TOC automatically and it recommends including all headers in the TOC, so I've stuck with that.

Reply:
I've used that extension before. You can specify the levels in the VS Code's JSON settings as follows. In this case, you would only be showing level-2 to level-4 headings, effectively skipping the `# Schematic` level-1 heading. This can wait until you next update the README. 

```
"markdown.extension.toc.githubCompatibility": true,
"markdown.extension.toc.levels": "2..4",
```